### DISCUSSION TITLE
How to disable Revolt status on Discord?

### DISCUSSION THREAD

**Discussion by thecauagames** [stoatchat/discussions#430] [Q&A] [answered]
Hello! I'm new in here and, every time I start Revolt, it appears on my Discord status, is there any way I can disable this? The 'Enable Discord status' on Revolt is disabled and still they show on Discord (My english is somewhere bad because it isn't my native language)

Here the option disabled on Revolt:
![Revolt setting](https://user-images.githubusercontent.com/83296719/190463238-508e0197-6b2c-495a-a950-0d3b039466a4.PNG)

And here Discord showing Revolt as my status:
![Discord showing Revolt](https://user-images.githubusercontent.com/83296719/190463396-b7ba2185-dca1-4ee2-a35e-8a4c623e7966.PNG)

**Comment by Rexogamer** [comment, marked as answer]
This is technically a Discord issue - it automatically detects Revolt as Re-Volt (a racing game from the late 90s) because the file name is `Revolt.exe`. The setting you disabled determines whether to use our rich presence (which looks like this):

![Screenshot of Revolt's rich presence on Discord - there is a red QR code that leads to the Revolt website, and two buttons ("Join Public Beta" and "Website".)](https://user-images.githubusercontent.com/42586271/190470037-647be3f8-98da-4c17-bfbd-e66f8b8e15eb.png)

If you hover over the listing on Discord, you should see this X button:

![Screenshot of the "Registered Games" page in Discord's settings - one app, Revolt, is visible and listed as "now playing". There are two buttons - a visible toggle (resembling a monitor with a slash through it) and a button to remove the app (a circular button with a red X).](https://user-images.githubusercontent.com/42586271/190470502-eb45650f-88d3-4047-b693-7c265315ad04.png)

Click it and Revolt will no longer show up in your status. I hope this helps! ^^

**Reply by thecauagames** [reply]
Hi, I'm back! If I rename the `Revolt.exe` to `Revolte.exe` for example, does Discord still thinks it's this game? Sadly for me isn't appearing the X button to remove this Re-Volt game of my Discord status, it appears the verified mark and enable overlay button

**Reply by Rexogamer** [reply]
That's odd - can you share a screenshot/recording of you hovering over the game?

To answer your first question, yes - renaming the file to, say, `RevoltDesktop.exe` would prevent Discord from assuming it's Re-Volt.

**Reply by thecauagames** [reply]
Sorry for the late reply, but here the screenshot:
![Discord screenshot](https://user-images.githubusercontent.com/83296719/190707285-9ba8d7c3-d077-4e13-8cc0-4af6cbcceed1.png)
Doesn't appear the X button to remove, only the verified mark as I said in the other reply

**Reply by insertish** [reply]
Discord doesn't let you remove verified games, nothing we can do about that.
You could try a different filename again maybe it'll not detect it as such.

**Reply by thecauagames** [reply]
Hi, sorry for the late reply (again), I changed the filename and the Re-Volt game status on Discord was no longer there! Thanks!

**Reply by ktwrd** [reply]
Created a merge request to fix this issue.
[revoltchat/desktop#121](https://github.com/revoltchat/desktop/pull/121)

**Reply by Andre601** [reply]
> Created a merge request to fix this issue. [revoltchat/desktop#121](https://github.com/revoltchat/desktop/pull/121)

I feel like this isn't the best way to go here.
As pointed out is this more a Discord issue, with them having a relatively stupid way of detecting "games" based on executable names.
So this should preferably be reported to them, as otherwise this issue may persist for any future programm that shares a name with another programm/game (And it can even be seen as a possible phishing/scam option as someone could run an exe to display Rich presence stuff as a "legit" game... Tho not sure if that would be actually possible here...)
